### PR TITLE
Add rnasync cron job to dev

### PR DESCRIPTION
**Pull request by jgmize**

### CODE REVIEW

**Comment by jgmize** [comment]
@pmclanahan r?

File: etc/cron.d/bedrock-dev.tmpl [hunk 1]
```diff
@@ -7,6 +7,7 @@ MAILTO="webops-cron@mozilla.com,cron-bedrock@mozilla.com"
 # add date/time info to cron log
 0    * * * * {{ user }} date {{ log }}
 
+*/6  * * * * {{ django_manage }} rnasync {{ log }}
```
Comment (pmclanahan) [inline etc/cron.d/bedrock-dev.tmpl]: Why do we need RNA data synced every 6 minutes on dev?

Comment (jgmize) [inline etc/cron.d/bedrock-dev.tmpl]: The release managers use the dev server to preview release notes.

Comment (jgmize) [inline etc/cron.d/bedrock-dev.tmpl]: https://github.com/mozilla/bedrock/blob/master/bedrock/releasenotes/views.py#L62

Comment (pmclanahan) [inline etc/cron.d/bedrock-dev.tmpl]: Ah. Cool.

**Pull request merged**